snippets/js/create-client-instance/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
It looks like all of the source files are not building properly, and the build folder doesn't actually get generated. We should be careful with these snippets to make sure they're all building properly, that way we don't have "dead code" that we think is functional but actually isn't being built / run.

I'm pushing an update now to fix this.